core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalSystem.java

```diff
@@ -137,6 +145,31 @@ public CatchupFuture catchup(Map<String, Long> journalSequenceNumbers) throws IO
     return CatchupFuture.allOf(futures);
   }
 
+  @Override
+  public void waitForInitialReplay() {
+    long start = System.currentTimeMillis();
+    try {
+      CommonUtils.waitFor("journal replay to finish catching up", () -> {
+        for (UfsJournal journal : mJournals.values()) {
+          UfsJournalCheckpointThread.ReplayState replayState = journal.getReplayState();
+          if (replayState != UfsJournalCheckpointThread.ReplayState.REPLAY_DONE) {
+            // keep flushing entries
```

Comment:
I don't think this is "flushing" the journal, but replaying the journal, right?

Reply:
Removed the comment